src/schema/core.yaml

```diff
@@ -74,6 +74,29 @@ types:
     
 classes:
 
+  data object:
+    is_a: named thing
+    in_subset:
```

Comment:
I am pretty sure a `subsets` section will need to be added to `core.yaml` or even the the `subset` may be deleted. For example, here is the `subsets` section in `nmdc.yaml`:
```yaml
subsets:
  sample subset:
    description: >-
      Subset consisting of entities linked to the processing of samples. 
      Currently, this subset consists of study, omics process, and biosample.
  
  data object subset:
    description: >-
      Subset consisting of the data objects that either inputs or outputs of processes or workflows.
```

Reply:
yeah, moved this in a commit while you were drafting this comment. :)